swarm/api/http/server.go

```diff
@@ -890,8 +913,17 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Header().Set("Content-Type", contentType)
-	http.ServeContent(w, r, "", time.Now(), newBufferedReadSeeker(reader, getFileBufferSize))
+	if contentType != "" {
+		w.Header().Set("Content-Type", contentType)
+	}
+
+	fileName := uri.Addr
+	if found := path.Base(uri.Path); found != "" && found != "." && found != "/" {
+		fileName = found
+	}
+	w.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", fileName))
```

Comment:
Are you sure we want to make all files attachments? If I go to `bzz://mysite.eth/index.html` in my browser, I expect to see the HTML in the browser, not download it to a local file.

Reply:
oh, how much you right. Right now i will change it to:
`Content-Disposition: inline; filename="example.html"`
But i will rise question in Gitter - maybe exist some cases when `attachment` make sense, for example in S3 owner can add any headers to it's file/folder by UI, and `Content-Disposition: attachment` too.